test/lib/index.test.js

```diff
@@ -7,18 +7,23 @@ const libingester = require('../../lib/index');
 describe('ImageAsset', function() {
     it('can serialize out correctly', function() {
         const asset = new libingester.ImageAsset();
+        const thumbnail_asset = new libingester.ImageAsset();
         asset.set_title('Test Asset');
         asset.set_synopsis('Test Asset synopsis');
+        asset.set_thumbnail(thumbnail_asset);
         asset.set_license('Proprietary');
         asset.set_canonical_uri('https://www.example.com/');
         asset.set_last_modified_date(new Date(1492545280000));
         asset.set_image_data('image/jpeg', 'asdf');
 
         const metadata = asset.to_metadata();
 
-        // Remove randomness -- should probably be a mock if I can
-        // figure out how to use it.
+        // Check that asset ID and thumbnail asset ID are passed through
+	expect(metadata['assetID']).to.equal(asset.asset_id);
+	expect(metadata['thumbnail']).to.equal(thumbnail_asset.asset_id);
+	// Remove the ID fields before checking the rest
         delete metadata['assetID'];
+        delete metadata['thumbnail'];
```

Comment:
@lmyllari Indeed - this is good. I don't see any tests for other types of assets but we can deal with that later.